packages/cart-sdk/package.json

```diff
@@ -7,9 +7,10 @@
    "dependencies": {
       "@ifixit/analytics": "workspace:*",
       "@ifixit/app": "workspace:*",
+      "@ifixit/auth-sdk": "workspace:*",
       "@ifixit/helpers": "workspace:*",
       "@ifixit/ifixit-api-client": "workspace:*",
-      "@ifixit/auth-sdk": "workspace:*",
+      "@ifixit/local-storage": "workspace:*",
```

Comment:
We wanted to use this code in `packages/` and `frontend/`. This could have been added to an existing package (like `helpers` maybe), but I'm not aware of any disadvantages to this approach so I thought this was fine.

Reply:
> but I'm not aware of any disadvantages to this approach so I thought this was fine

To evaluate the cost of things like this, I think about expanding a pattern. Would I want every single file to be turned into a package that each has it's own package.json? In my opinion the answer there is no because it's more boilerplate, more futzing with extra files when adding an `import ...` statement (must declare dependencies in two places), and likely slower installs.

I think there is utility in some amount of separation of responsibilities, but in cases like this (a thin safety wrapper around a built-in) I don't think there's any gains in separating it into a new package.

Reply:
I agree with you, and I don't think I would have made this a new package if I had implemented it. At the same time, there is a cost to another round of CR/QA and I chose to accept this working solution at the time.

Reply:
> there is a cost to another round of CR/QA and I chose to accept this working solution at the time.

I agree, let's not open a new pull.

Reply:
> is there a disadvantage to adding a new package?

Not very much for a single new package, but I enumerated the reasoning in my [previous comment](https://github.com/iFixit/react-commerce/pull/1817#discussion_r1265722416)

Reply:
is there a place where in an existing package that you think a localstorage wrapper logically belongs in? because to me i think having a sensible/readable file hierarchy is valuable too

Reply:
I'd probably vote for the `helpers` package, but it doesn't matter that much.